CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 Changes
 =======
+## 6.1.0 / 2024-04-23
+https://github.com/jenkinsci/datadog-plugin/compare/datadog-6.0.3...datadog-6.1.0
+
+* [Changed] Changed plugin to Java 11. See [#404](https://github.com/jenkinsci/datadog-plugin/pull/404)
```

Comment:
```suggestion
* [Added] Changed plugin to Java 11. See [#404](https://github.com/jenkinsci/datadog-plugin/pull/404)
```
nit- a `changed` entry in a changelog typically corresponds to a major version bump, ie 7.0.0